statistician/Dockerfile

```diff
@@ -40,10 +40,11 @@ RUN apt-get update \
 # Add in the dask configuration
 COPY distributed.yaml /etc/dask/distributed.yaml
 
-COPY requirements.txt constraints.txt version.txt /conf/
+COPY setup_reqs.txt requirements.txt constraints.txt version.txt /conf/
 
 RUN cat /conf/version.txt \
   && pip install --no-cache-dir --upgrade pip \
+  && pip install --no-cache-dir -r /conf/setup_reqs.txt\
```

Comment:
Why are they in a separate `setup_reqs.txt` file instead of in `requirements.txt`?

Reply:
it's about how `pip install -r` works. `pip install -r` won't install all the things in the requirements sequentially. It's gathers everything and sorta install things in parallel. So if a package doesn't specify the setup requirements in its`setup.py`, `pip install` won't collect the required packages first and install them, which means in this case, whatever is required has to be installed ahead. For this instance, one of required package doesn't have a `setup_requires` in their `setup.py` file. Hence they're done in a separate requirement.

Reply:
Which package is the naughty one that doesn't specify it's `setup_requires` properly?

Reply:
`morecantile` required by `rio-cogeo`